skflow/__init__.py

```diff
@@ -67,6 +67,13 @@ def __init__(self, model_fn, n_classes, tf_master="", batch_size=32, steps=50, o
         self.num_cores = num_cores
         self._initialized = False
 
+    def _data_type_filter(self, X, y):
```

Comment:
I think it makes sense somewhere around data feeders, but they are not used right now for predict... We can keep it here for now and then move as more variety of io stuff will be needed.